xbmc/input/joysticks/generic/GenericJoystickActionHandler.cpp

```diff
@@ -117,7 +117,8 @@ void CGenericJoystickActionHandler::OnDigitalAction(unsigned int buttonId, unsig
   newEvent.joystick.button = buttonId;
   newEvent.joystick.holdtime = holdTimeMs;
 
-  CWinEvents::MessagePush(&newEvent);
+  //CWinEvents::MessagePush(&newEvent);
+  g_application.OnEvent(newEvent);
```

Comment:
CApplication::OnEvent() is static, so this would be

`CApplication::OnEvent(newEvent);`